### PR TITLE
Upgrade Origami dependencies

### PR DESCRIPTION
**Pull request by sumeetadur**

### CODE REVIEW

**Comment by sumeetadur** [comment]
@kavanagh / @callumlocke can you peer review this?

File: .gitignore [hunk 1]
```diff
@@ -4,3 +4,5 @@
 /.tmp/
 /.sass-cache/
 .DS_Store
+
```
Comment (kavanagh) [inline .gitignore]: line break

File: client/index.html [hunk 2]
```diff
@@ -25,11 +25,11 @@
     <!-- endsmoosh -->
 
     <!-- o-grid css -->
-    <link rel="stylesheet" href="https://build.origami.ft.com/v2/bundles/css?modules=o-grid@^4.0.0" />
+    <link rel="stylesheet" href="https://build.origami.ft.com/v2/bundles/css?modules=o-grid@^4.0.7" />
     <!-- end o-grid css -->
 
     <!-- origami build service -->
-    <link rel="stylesheet" href="https://origami-buildservice-qa.herokuapp.com/bundles/css?modules=o-forms@^2.0.0">
```
Comment (kavanagh) [inline client/index.html]: great origami docs corrected

File: bower.json [hunk 3]
```diff
@@ -2,12 +2,12 @@
   "name": "origami-product-boilerplate",
   "private": true,
   "dependencies": {
-    "normalize.css": "~3.0.2",
-    "o-colors": ">=2.5.0 <4",
-    "o-hoverable": "~1.1.0",
-    "o-fonts": "~1.6.4",
+    "normalize.css": "~3.0.3",
```
Comment (kavanagh) [inline bower.json]: ~ or ^ ?

Comment (sumeetadur) [inline bower.json]: ~ to match the starter kit dependencies before the move to n-normalize

**Pull request merged**